x/stakeibc/keeper/icqcallbacks_withdrawal_balance.go

```diff
@@ -37,14 +37,14 @@ func WithdrawalBalanceCallback(k Keeper, ctx sdk.Context, args []byte, query icq
 	// Check if the coin is nil (which would indicate the account never had a balance)
 	if withdrawalBalanceCoin.IsNil() || withdrawalBalanceCoin.Amount.IsNil() {
 		k.Logger(ctx).Info(fmt.Sprintf("WithdrawalBalanceCallback: balance query returned a nil coin for address %s on %s, meaning the account has never had a balance on the host",
-			hostZone.WithdrawalAccount.Address, hostZone.ChainId))
+			hostZone.WithdrawalAccount.GetAddress(), hostZone.ChainId))
```

Comment:
Should we make all the attribute references getters, across the codebase? E.g. hostZone.GetChainId()

Reply:
chatted offline, only makes sense to use these for error logs where the state of the logged object is unknown